canonical-web-and-design/jaas-dashboard
# Add basepath to router

## Done

Add basepath to router.

## QA

- Pull code
- Modify `REACT_APP_BASE_APP_URL` in `.env` to be `/purple`
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/purple
- Navigate around the app, it should work as expected.

## Details

Fixes #279
